source/core/parameters.cc

```diff
@@ -1920,6 +1920,13 @@ namespace Parameters
         "Enable testing mode of a solver. Some solvers have a specific"
         "testing mode which enables the output of debug variables. This"
         "testing mode is generally used only for the automatic testing bench using ctest.");
+      prm.declare_entry(
+        "type",
```

Comment:
Because in the future we could use it for any data (void fraction, granular temperature etc), so I kept it general even if there's no other option at the moment